etc/rose-meta/rose-suite-info/rose-meta.conf

```diff
@@ -62,11 +66,13 @@ sort-key=00-type-0
 type=meta
 
 [=sub-project]
+compulsory=true
```

Comment:
Existing suites won't validate with this change, so I again don't think this should be compulsory.